editor/editor_build_profile.cpp

```diff
@@ -597,7 +597,7 @@ void EditorBuildProfileManager::_fill_classes_from(TreeItem *p_parent, const Str
 	TreeItem *class_item = class_list->create_item(p_parent);
 	class_item->set_cell_mode(0, TreeItem::CELL_MODE_CHECK);
 	class_item->set_icon(0, EditorNode::get_singleton()->get_class_icon(p_class));
-	String text = p_class;
+	const String &text = p_class;
```

Comment:
Same here

Reply:
Same here. The `text = <...>` pattern is quite common within file. I'm biased against this change since I think it adds code patterning and readability for the maintainer.